GTT.Database/dbo/Tables/Classes.sql

```diff
@@ -0,0 +1,4 @@
+﻿CREATE TABLE [dbo].[Classes] (
```

Comment:
As discussed, we need to keep this project up-to-date with latest DB. Please remove unused tables and sync latest DB back to this project's DB objects